bool enabled
float64 distanceMin
float64 distanceMax
float64 speedMin
float64 speedMax
float64 angleMin
float64 angleMax
string  angleTest
bool    angleTestBilateral
float64 timeHold
float64 timeout


